src/components/Chat/MessageAttachments.tsx

```diff
@@ -35,13 +35,13 @@ export const MessageAttachments = ({ attachments, isInner }: Props) => {
         <ChevronDown
           height={18}
           width={18}
-          className={`shrink-0 text-gray-500 transition ${
+          className={`shrink-0 text-secondary transition ${
             isSectionOpened ? 'rotate-180' : ''
           }`}
         />
       </button>
       {isSectionOpened && (
-        <div className="grid max-w-full grid-cols-1 gap-1 border-t border-gray-400 p-2 dark:border-gray-700 sm:grid-cols-2 md:grid-cols-3">
+        <div className="grid max-w-full grid-cols-1 gap-1 border-t border-primary p-2 sm:grid-cols-2 md:grid-cols-3">
```

Comment:
```suggestion
        <div className="grid max-w-full grid-cols-1 gap-1 border-t border-secondary p-2 sm:grid-cols-2 md:grid-cols-3">
```
![image](https://github.com/epam/ai-dial-chat/assets/20163971/62cf7611-aa86-4ea4-b2f8-55d321eb78ee)